### DISCUSSION TITLE
Towny Data Corruption? Safe Mode enabled, need urgent help

### DISCUSSION THREAD

**Discussion by Danieltjy1186** [TownyAdvanced/Towny#6523] [Q&A] [answered]
Towny Data Corruption? Safe Mode enabled, need urgent help

I tried what it said on the website for the safe mode guide for towny, and it doesn't really work. this is my 3rd tar.gz latest log file that i am unzipping and seeing it's still corrupted.

**Comment by LlmDl** [comment, marked as answer]
File a bug report and include your startup. 

https://github.com/TownyAdvanced/Towny#a-special-message-from-llmdl-precedes-the-normal-readme-
